BUILDING.md

```diff
@@ -0,0 +1,68 @@
+# Building JanusGraph client for Python
+
+- Building Library:
+
+A set of scripts have been created to make the job of building the library easier for user, but the scripts
+expects few pre requisites to be installed before hand. 
+
+- Pre-Requisites:
+    - Python needs to be installed on system. The library is currently tested against Python
+    3.4, 3.5 & 3.6, though is expected to work against any Python 3.
+    
+    - Python is added to system path. Make sure, the user can run the following commands from command line:
+        - python3/ python3.4/ python3.5/ python3.6 on Unix machines.
+        - python on Windows machines.
+    
+    - The automated build scripts are tested on following Operating Systems:
+        - Windows 7
+        - CentOS
+        - RHEL 7
+        - Ubuntu 18.04
+        - MacOS
+        
+- Building on Windows 7:
+    - A script build.cmd is created which accepts following keyword arguments:
+        - d: Specifies weather documents to be build. Set to true or false.
+        - b: Specifies weather library to be build. Set to true or false.
+        - i: Specifies weather library to be installed or not. Set to true or false.
+    
+    - If any of parameters is missing, the script exits with error message. 
+    An appropriate way to invoke the script would be:
+        ```bash
+        build.cmd "d=true" "b=true" "i=true"
+        ```
+    - The above example builds the docs, library and also installs to local global Python site-packages.
+
+- Building on Unix System:
+    - A script named build.sh is created which accepts following parameters:
+        - -d: Default to true. Implies, if documentation is to be build.
```

Comment:
Put `-d` in backticks so that it's formatted as code; similarly for the flags below.